public/data/javascript.json

```diff
@@ -910,33 +910,6 @@
         "tags": ["javascript", "function", "compose", "utility"],
         "author": "axorax"
       },
-      {
-        "title": "Rate Limit Function",
```

Comment:
The `Random string` is a duplicate, this one was fine... Sorry i should have included the name of the snippet